packages/docusaurus-theme-classic/codeTranslations/ar.json

```diff
@@ -30,29 +30,29 @@
   "theme.blog.post.plurals": "مقاله واحده|{count} مقالات",
   "theme.blog.post.readMore": "اقرأ المزيد",
   "theme.blog.post.readingTime.plurals": "{readingTime} دقائق قراءة|{readingTime} دقائق قراءة",
-  "theme.blog.sidebar.navAriaLabel": "Blog recent posts navigation",
+  "theme.blog.sidebar.navAriaLabel": "أحدث مشاركات المدونة",
   "theme.blog.tagTitle": "{nPosts} موسومة ب \"{tagName}\"",
   "theme.common.editThisPage": "تعديل هذه الصفحة",
   "theme.common.headingLinkTitle": "ارتباط مباشر بالعنوان",
   "theme.common.skipToMainContent": "انتقل إلى المحتوى الرئيسي",
-  "theme.docs.paginator.navAriaLabel": "التنقل بين صفحات المستندات",
+  "theme.docs.paginator.navAriaLabel": "التنقل بين صفحات الددات",
   "theme.docs.paginator.next": "التالى",
   "theme.docs.paginator.previous": "السابق",
   "theme.docs.sidebar.collapseButtonAriaLabel": "طي الشريط الجانبي",
   "theme.docs.sidebar.collapseButtonTitle": "طي الشريط الجانبي",
   "theme.docs.sidebar.expandButtonAriaLabel": "توسيع الشريط الجانبي",
   "theme.docs.sidebar.expandButtonTitle": "توسيع الشريط الجانبي",
-  "theme.docs.tagDocListPageTitle": "{nDocsTagged} with \"{tagName}\"",
-  "theme.docs.tagDocListPageTitle.nDocsTagged": "One doc tagged|{count} docs tagged",
+  "theme.docs.tagDocListPageTitle": "{nDocsTagged} مستند موسوم بـ \"{tagName}\"",
+  "theme.docs.tagDocListPageTitle.nDocsTagged": "مستند موسوم واحد|{count} مستندات موسومة",
   "theme.docs.versions.latestVersionLinkLabel": "احدث اصدار",
   "theme.docs.versions.latestVersionSuggestionLabel": "للحصول على أحدث الوثائق، راجع {latestVersionLink} ({versionLabel}).",
   "theme.docs.versions.unmaintainedVersionLabel": "هذه هي وثائق {siteTitle} {versionLabel}، التي لم تعد تتم صيانتها بشكل نشط.",
-  "theme.docs.versions.unreleasedVersionLabel": "هذه وثائق لم يتم طرحها لإصدار {siteTitle} {versionLabel}.",
+  "theme.docs.versions.unreleasedVersionLabel": "هذه وثائق لم يتم طرحها للإصدار {siteTitle} {versionLabel}.",
   "theme.lastUpdated.atDate": " في {date}",
   "theme.lastUpdated.byUser": " بواسطة {user}",
-  "theme.lastUpdated.lastUpdatedAtBy": "آخر تحديث{atDate}{byUser}",
-  "theme.navbar.mobileSidebarSecondaryMenu.backButtonLabel": "← Back to main menu",
-  "theme.navbar.mobileVersionsDropdown.label": "Versions",
+  "theme.lastUpdated.lastUpdatedAtBy": "آخر تحديث في {atDate} بواسطة {byUser}",
```

Comment:
We don't ue spaces here on purpose around `{atDate}{byUser}` , because the spaces are already contained in `theme.lastUpdated.atDate` and  `"theme.lastUpdated.byUser`

Adding spaces around those placeholders would lead to duplicate spaces

Reply:
OK, I will update the files